From bfdadee5e82155bf25eba5a5e841f8029e6d7daa Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Sat, 27 Oct 2018 13:42:13 +0100
Subject: [PATCH 20/29] overlay: Hide the option to show battery percentage

 * The option is available in Status bar settings,
   as per our additions. Remove duplicate to improve UX.

Change-Id: I60a737617f192f5f0699ce50cfc3edc52c862492
---
 overlay/common/frameworks/base/core/res/res/values/config.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/overlay/common/frameworks/base/core/res/res/values/config.xml b/overlay/common/frameworks/base/core/res/res/values/config.xml
index 9b135bb7..86e2a2f1 100644
--- a/overlay/common/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/common/frameworks/base/core/res/res/values/config.xml
@@ -92,4 +92,7 @@
 
     <!-- Whether or not swipe up gesture's opt-in setting is available on this device -->
     <bool name="config_swipe_up_gesture_setting_available">true</bool>
+
+    <!-- Whether or not we should show the option to show battery percentage -->
+    <bool name="config_battery_percentage_setting_available">false</bool>
 </resources>
-- 
2.17.1

